sysl2/sysl/seqs.go

```diff
@@ -110,6 +122,12 @@ func DoConstructSequenceDiagrams(
 					log.Errorf("No call statements to build sequence diagram for endpoint %s", endpoint.Name)
 					return result
 				}
+				groupAttr := epAttrs["groupby"].GetS()
+				if len(groupAttr) == 0 {
+					groupAttr = group
+				} else if len(group) > 0 {
+					log.Warnf("Ignoring groupby passed from command line")
```

Comment:
i suspect we might want to give user more control in the future i.e. override the group by value from command line.